bml/lib/data_summary.py

```diff
@@ -59,37 +63,42 @@ def print_run_details(config, es_backend, uuid, update):
         test_name = test_type['test']
         data = []
         test_run = None
+        check_outcome = 0  # just to see if it passses all conditions
```

Comment:
The idea was that if one of the conditions say test_check failed, then we'll handle it differently but then during testing I realized it might not be needed right now. Can change it to boolean, but would rather keep it this way for a couple of days and if it's working fine will change it to boolean.